### ISSUE TITLE
Paladin Tier 2 Judgement 8/8 set bonus dosent work as supposed

### ISSUE THREAD

**Issue by fidbugs** [elysium-project/server#614]
Currently Judgement Tier 2 set bonus 8/8 is not working properly. It scales with spell power and does not have ability to critically hit. Also it triggers after actual Judgement of Seal XY - combat first shows Judgementof "insert seal", then the Judgement as spell from T2 8/8 set bonus. 

https://gyazo.com/faeea2123190ab4c14d4c7c576a94b75 - current Anathema state

Ability is just Judgement, hit 196, 0 critical strikes and maximum of 159 damage dealt.

In video "Spell damage paladin in action" all thru the video you can see that damage of the Judgement does not go over 66. It does not scale with either spell damage from gear(wearing full T2) nor for Judgement of the Crusader(but it is affected by Berserker stance and Power Infusion). Also on 2:17 you can see Judgement critically hit for 122 damage, 200% of normal - that would strongly suggest that spell uses same mechanic as actual paladin Judgement spell for "to hit" and "to criticaly hit" atributes. To hit is not so important since it requires successful Judgement to land, but chance to critically hit is extremely important as it takes it directly from Judgement formula. 

https://youtu.be/ay7RU0N-44Y?t=137 - "Spell damage paladin in action"

http://web.archive.org/web/20071225172821/http://thottbot.com:80/s23591 - thotbot comment from user Reddimus stating it is not affected by spell power but can criticaly hit

https://www.youtube.com/watch?v=5vGxOFfo6As&feature=youtu.be&t=90 - another video where you can see it criticaly hit at 4:24 for 200% of normal, also log is seen regarding the order of spells

To recap what should be fixed:

a. Is not affected with bonus spell damage.
b. Has ability to be a critical hit, 200% of normal.
b1. Same mechanic as actual Judgement spell.
c. Should go in combat log before the actual Judgement of "certain seal".

**Comment by killerduki** [comment]
To clarify you b. point , according to the video you gave where it criticaly hit, we can see that it does 100% crit instead 50% crit, which could "assume" also scale of melee hit rating either.

**Comment by Mostprovidedlinks** [comment]
It should be affected by player spell crit. The commenter said he didn't know if it was affected by player spell crit but he said it crits. Its spell number (2) just like blazefury medallion, dark rune and demonic rune (affected by player spell damage and spell crit - I have a 2006 site) storm gauntlets, fiery retributer, fiery plate gauntlets, all scale with spell crit and indicate it should be affected by spell crit.

**Comment by killerduki** [comment]
@Mostprovidedlinks  probably, still on the Video we all saw that it does 100% crit , not 50%.

**Comment by fidbugs** [comment]
@Mostprovidedlinks It would be more likely that it uses same to hit/ to critically hit mechanic as actual Judgement spell, as @killerduki noted it actually hits for 200% of normal so that also gives weight to claim that it has same mechanic as Judgement spell. What database says about spell used could be misleading and showing only partial mechanic of ability.

**Comment by Mostprovidedlinks** [comment]
I agree with them
